### ISSUE TITLE
Cannot build

### ISSUE THREAD

**Issue by ghost** [etil2jz/Mirai#3]
Hi! I'm trying to build Mirai on my local PC (Windows 11 with WSL) and getting this error:
```gradle
pelfox@pelfox-dev:~/Mirai$ ./gradlew applyPatches
To honour the JVM settings for this build a single-use Daemon process will be forked. See https://docs.gradle.org/7.3.1/userguide/gradle_daemon.html#sec:disabling_the_daemon.
Daemon will be stopped at the end of the build
Configuration on demand is an incubating feature.

> Task :Pufferfish:paper:applyServerPatches
Auto packing the repository in background for optimum performance.
See "git help gc" for manual housekeeping.

> Task :applyApiPatches FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':applyApiPatches' (type 'SimpleApplyGitPatches').
  - In plugin 'io.papermc.paperweight.patcher' type 'io.papermc.paperweight.patcher.tasks.SimpleApplyGitPatches' property 'upstreamDir' specifies directory '/home/pelfox/Mirai/.gradle/caches/paperweight/upstreams/Pufferfish/Pufferfish-API' which doesn't exist.

    Reason: An input file was expected to be present but it doesn't exist.

    Possible solutions:
      1. Make sure the directory exists before the task is called.
      2. Make sure that the task which produces the directory is declared as an input.

    Please refer to https://docs.gradle.org/7.3.1/userguide/validation_problems.html#input_file_does_not_exist for more details about this problem.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 52s
3 actionable tasks: 3 executed
```

**Comment by ghost** [comment]
See [this](https://github.com/pufferfish-gg/Pufferfish/issues/2) issue to get information how to fix this error. I'll try to do it myself, but later.

**Comment by etil2jz** [comment]
Since #5, vanilla class import doesn't work, which breaks patch applying as some classes are imported manually. I reopen this issue.

**Comment by etil2jz** [comment]
@Pelfox Fixed since commit f9fcb70 😉
